github/Repository.py

```diff
@@ -1836,15 +1840,20 @@ def create_deployment(
         payload=github.GithubObject.NotSet,
         environment=github.GithubObject.NotSet,
         description=github.GithubObject.NotSet,
+        transient_environment=github.GithubObject.NotSet,
+        production_environment=github.GithubObject.NotSet,
     ):
         """
         :calls: `POST /repos/:owner/:repo/deployments <https://developer.github.com/v3/repos/deployments/>`_
         :param: ref: string
+        :param: task: string
         :param: auto_merge: bool
-        :param: required_contexts: list of statuses
+        :param: required_contexts: list of status contexts
         :param: payload: json
```

Comment:
This is no longer json

Reply:
Further more, we need to remember to mention the API break on the next release.

Reply:
This is now implemented.